src/main/resources/migration/changelog-v5.5.0.xml

```diff
@@ -110,4 +110,27 @@
             <column name="OPERATION_MODE" type="VARCHAR(255)"/>
         </addColumn>
     </changeSet>
+
+    <changeSet id="v5.5.0-10" author="sahibamittal">
+        <createTable tableName="COMPONENT_PROPERTY">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="COMPONENT_PROPERTY_PK"/>
+            </column>
+            <column name="COMPONENT_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
There should be a [foreign key constraint](https://docs.liquibase.com/change-types/add-foreign-key-constraint.html) for this column.

I'd vote for us to leverage `onDelete="CASCADE"` going forward. Component properties can't exist without their owning component, so when that component is deleted, the database should take care of also deleting the associated properties.